modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
     Tls          On
 ```
 
+==== Configure Log Forwarding
+
+The following configuration options are available:
+
+* Mule Applications: Forward logs from Mule applications.
+* Internal load balancer and component logs: Forward logs from Runtime Fabric namespace compenents and internal load balancers.
+* Runtime Fabric appliance: Forward logs from Anypint Monitoring and Kube-system namespaces.
+
+[NOTE]
+====
+* By default all options on Forward logs configuration are checked.
+* Configuring Forward logs from Mule applications is not currently supported, so this option is checked and disabled. You can still enable or disable per application log forwarding from Applications page.
```

Comment:
"Disabling log forwarding for Mule applications is not supported. Log forwarding can be disabled per application during deployment."